plugins/Installation/tests/UI/Installation_spec.js

```diff
@@ -18,6 +18,7 @@ describe("Installation", function () {
         testEnvironment.testUseMockAuth = 0;
         testEnvironment.configFileLocal = path.join(PIWIK_INCLUDE_PATH, "/tmp/test.config.ini.php");
         testEnvironment.dontUseTestConfig = true;
+        testEnvironment.ignoreClearAllViewDataTableParameters = 1;
```

Comment:
Should it default to 0?

Reply:
It defaults to not being set. Various UI tests are reusing the same database (to speed things up), so we need to clean up the changes that might be store to views automatically. Otherwise tests would have different results depending which tests ran before. To prevent that this variable (previously get parameter) can be provided.
In that specific case we can't try to clean up as the database was not yet created in installation tests.